core/commands/log_graph.py

```diff
@@ -17,10 +17,12 @@ def run(self):
         view = self.window.new_file()
         view.settings().set("git_savvy.log_graph_view", True)
         view.settings().set("git_savvy.repo_path", repo_path)
+        view.set_syntax_file("Packages/GitSavvy/syntax/graph.tmLanguage")
         view.set_name(LOG_GRAPH_TITLE)
         view.set_scratch(True)
         view.set_read_only(True)
         view.run_command("gs_log_graph_initialize")
+        view.sel().clear()
```

Comment:
Instead of adding this line here, modify the last line of `GsLogGraphInitializeCommand#run` to the following:
```python
self.view.run_command("gs_replace_view_text", {"text": branch_graph, "nuke_cursors": True})
```